drivers/clock_control/Kconfig.pwm

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 Andriy Gelman <andriy.gelman@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config CLOCK_CONTROL_PWM
+	bool "Generic PWM clock"
+	default y
+	depends on DT_HAS_PWM_CLOCK_ENABLED
+	select PWM
+	help
+	  Enable generic PWM clock.
+
+if CLOCK_CONTROL_PWM
+
+config CLOCK_CONTROL_PWM_INIT_PRIORITY
+	int "Initialization priority of the pwm clock device"
+	default 51
+	help
+	  Initialization priority of the pwm clock device. Must be
+	  lower priority than pwm.
+
+config CLOCK_CONTROL_PWM_ON_DELAY_USEC
```

Comment:
This is a hardware property. It should go into DTS.

Reply:
thanks, will update.